### DISCUSSION TITLE
[explore] add chart to a selected tab on save to dashboard

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#18439] [Ideas]
"It would be great if I can choose which tab to add a chart to dashboard to rather than dropped in the first page."
 And “nice-to-have” choose “add to top” or “add to bottom.

<img width="601" alt="Screen Shot 2021-02-08 at 11 06 21 AM" src="https://user-images.githubusercontent.com/67837651/107269307-45e24b00-69fe-11eb-95a4-1eaec53445c3.png">

**Comment by srinify** [comment]
Love this and strongly agree! Having worked on at least 10, multi-tab dashboards .. being forced to hunt down the tab the chart dropped into and drag it into the tab I actually care about is an annoying pattern!

**Comment by mihir174** [comment]
Yup I think this is a great idea. I also figured if we do the bonus of adding to the bottom or top of a tab, we could provide the same option for adding to a dashboard in general

<img width="1408" alt="Slice 2 (2)" src="https://user-images.githubusercontent.com/64227069/107700821-e3db4d00-6c6c-11eb-9c9e-4a30ada66adb.png">
